PyStemmusScope/forcing_io.py

```diff
@@ -22,22 +22,32 @@ def _write_matlab_ascii(fname, data, ncols):
     np.savetxt(fname, data, multi_fmt)
 
 
-def read_forcing_data(forcing_file):
+def read_forcing_data(forcing_file, start_time, end_time):
     """Reads the forcing data from the provided netCDF file, and applies the required
     unit conversions before returning the read data.
 
     Args:
         forcing_file (Path): Path to the netCDF file containing the forcing data
+        start_time (str): Start time of time range.
+        end_time (str): Start time of time range.
 
     Returns:
         dict: Dictionary containing the different variables required by STEMMUS_SCOPE
             for the different forcing files.
     """
-    ds_forcing = xr.open_dataset(forcing_file)
+    ds_forcing = xr.open_dataset(forcing_file)    
```

Comment:
Trailing whitespace :)
```suggestion
    ds_forcing = xr.open_dataset(forcing_file)
```